src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporter.cs

```diff
@@ -104,21 +104,6 @@ internal void SetResource(Resource resource)
                 }
             }
 
-            if (!processResource.Attributes.Any(kvp => kvp.Key == Resource.ServiceNameKey))
```

Comment:
Shouldn't we keep the current behavior of falling back to DefaultServiceName?

Reply:
Ah ok makes sense, wasn't sure about that. Will make the change first thing tomorrow.